core/code/entity_decode.js

```diff
@@ -60,10 +60,20 @@ function parseArtifactDetail(arr) {
   };
 }
 
+function parseHistoryDetail(bitarray) {
+  if (bitarray == null) { return null; }
+  return {
+    _raw: bitarray,
+    visited:  !!(bitarray & (1|2|4)),
+    captured: !!(bitarray & 2),
+    scanned:  !!(bitarray & 4),
```

Comment:
```suggestion
    scout-controlled:  !!(bitarray & 4),
```
The bit flag here does not indicate that a portal has been scanned.  It indicates that the portal has been held as Scout Controller.  Using "scanned" is misleading.

Reply:
Attention!
At the moment 'scanned' layer is wrongly titled as 'Scout Controlled' in stock intel!

"Scout Controller" is a volatile status, that the agent looses sooner or later on interaction of other agents with the same portal.
"scanned" is a permanent status, set when the agent submits the first 3D-Scan of a portal.